es/refresher-linear-algebra.md

```diff
@@ -1,339 +1,341 @@
 
 <br>
 
 **35. N(ax)=|a|N(x) for a scalar**
 
-&#10230;
+&#10230; N(ax)=|a|N(x) para un escalar
 
 <br>
 
 **36. if N(x)=0, then x=0**
 
-&#10230;
+&#10230; si N(x)=0, entonces x=0
 
 <br>
 
 **37. For x∈V, the most commonly used norms are summed up in the table below:**
 
-&#10230;
+&#10230; Para x∈V, los modulos o normas más comunmente usadas están descritas en la tabla de abajo:
 
 <br>
 
 **38. [Norm, Notation, Definition, Use case]**
 
-&#10230;
+&#10230; [Norma, Notacion, Definición, Caso de uso]
 
 <br>
 
 **39. Linearly dependence ― A set of vectors is said to be linearly dependent if one of the vectors in the set can be defined as a linear combination of the others.**
 
-&#10230;
+&#10230; Dependencia Lineal ― Un conjunto de vectores se dice linearmente dependiente si uno de los vectores en el grupo puede ser definido como la combinación de los otros.
 
 <br>
 
 **40. Remark: if no vector can be written this way, then the vectors are said to be linearly independent**
 
-&#10230;
+&#10230; Nota: si no se puede escribir el vector de esta manera, entonces el vector se dice que es linealmente independiente
 
 <br>
 
 **41. Matrix rank ― The rank of a given matrix A is noted rank(A) and is the dimension of the vector space generated by its columns. This is equivalent to the maximum number of linearly independent columns of A.**
 
-&#10230;
-
+&#10230; Rango matricial ― El rango de una matriz A, nombrado rank(A), que es la dimensioón del vector espacial generado por sus columnas. Lo que es equivalente al número máximo de columnas linealmente independientes de A.
+ 
 <br>
 
 **42. Positive semi-definite matrix ― A matrix A∈Rn×n is positive semi-definite (PSD) and is noted A⪰0 if we have:**
 
-&#10230;
+&#10230; Matriz semi-definida positiva ― Una matriz A∈Rn×n es semi-defininda positivamente (PSD) y se tiene que A⪰0 si:
 
 <br>
 
 **43. Remark: similarly, a matrix A is said to be positive definite, and is noted A≻0, if it is a PSD matrix which satisfies for all non-zero vector x, xTAx>0.**
 
-&#10230;
+&#10230; Nota: de igual forma, una matriz A se dice positiva y definida, A≻0, si esa una matriz PSD que satisface para todos los vectores diferentes de cero x, xTAx>0.
 
 <br>
 
 **44. Eigenvalue, eigenvector ― Given a matrix A∈Rn×n, λ is said to be an eigenvalue of A if there exists a vector z∈Rn∖{0}, called eigenvector, such that we have:**
 
-&#10230;
+&#10230; Eigenvalor, eigenvector ― Dado una matriz A∈Rn×n, λ se dice que es el eigenvalor de A si existe un vector z∈Rn∖{0}, llamado eigenvector, tal que se tiene:
 
 <br>
 
 **45. Spectral theorem ― Let A∈Rn×n. If A is symmetric, then A is diagonalizable by a real orthogonal matrix U∈Rn×n. By noting Λ=diag(λ1,...,λn), we have:**
 
-&#10230;
+&#10230; Teoréma espectrasl ― Sea A∈Rn×n. si A es simétrica, entonces A es diagonalizable por una matriz real ortogonal U∈Rn×n.
```

Comment:
* Remove the accent from _Teorema_.
* You misspelt _espectral_ as "espectrasl".